python/pyspark/pandas/groupby.py

```diff
@@ -44,10 +43,16 @@
     TYPE_CHECKING,
 )
 
-import numpy as np
 import pandas as pd
 from pandas.api.types import is_hashable, is_list_like
 
+try:
+    from pandas.core.base import SelectionMixin
+
+    _builtin_table = SelectionMixin._builtin_table
+except AttributeError:
+    from pandas.core.common import _builtin_table
```

Comment:
@Cedric-Magnan, the mypy fails with:

```
python/pyspark/pandas/groupby.py:54: error: Name "_builtin_table" already defined on line 52
Found 1 error in 1 file (checked 315 source files)
```

Can we work around this?

Reply:
This should be solved now